content/post/074-explore-the-files-api-on-protoschool.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2019-10-29
+url: 2019-11-04-explore-the-files-api-on-protoschool
+title: Explore the Files API on ProtoSchool
+author: Teri Chadbourne
+---
+
+**TODO: FIX PUBLICATION DATE once finish previewing/editing**
+
+Today we're excited to announce the launch of a [brand new ProtoSchool tutorial exploring the **Regular Files API**](https://proto.school/#/regular-files-api), built by the awesome [Gil Domingues](https://github.com/dominguesgm). 🎉
```

Comment:
A nice screenshot/gif of the tutorial / protoschool website might ground people in what we're talking about / add some more content here